test/gtest/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
-find_package(GTest CONFIG REQUIRED)
+find_package(GTest CONFIG)
 if(GTest_FOUND)
   MESSAGE(STATUS "GoogleTest: " ${GTest})
-  enable_testing()
   include(GoogleTest)
+else()
+  message(WARNING "GoogleTest is not found! Skip GTests ... try install GTest or preferably follow instructions to install MIOpen dependencies.")
+  return()
```

Comment:
Is this change related to the PR?

Reply:
Not directly, a byproduct of the review I believe.